pkg/staticnetworkconfig/backward_compatibility.go

```diff
@@ -0,0 +1,5 @@
+package staticnetworkconfig
+
+const MinimalVersionForNmstatectl = "4.13"
```

Comment:
As discussed offline, we probably want '4.14' as the minimal supported version. This is due to an issue in nmstatectl with using 'identifier: mac-address' - which is resolved in version [2.2.35](https://github.com/nmstate/nmstate/releases/tag/v2.2.35).
This nmstate version should be backported to z-streams of rhcos>=4.14.

Reply:
According to Gris, version 2.2.35 will be released for RHEL 9.2 on October 1. It may take 1-2 weeks to see an OCP release that includes it. Since the mac-identifier has been broken in production for some time, we can continue working with version 4.13 until we receive the OCP versions from Gris. In addition, currently, all versions are affected by the mac-identifier issue anyway. what do you think?

Reply:
Ok, so if we don't know to which exact version the fix is going to land, then we can just change when needed.
I.e. if 4.13 won't get the fix, we can decide to not support it, or mention it in docs, etc.